sdk/python/pulumi_kubernetes/helm/v3/helm.py

```diff
@@ -555,91 +507,24 @@ def _parse_chart(all_config: Tuple[str, Union[ChartOpts, LocalChartOpts], pulumi
         fetch_opts_dict = {}
         if config.fetch_opts is not None:
             fetch_opts_dict = {k: v for k, v in vars(config.fetch_opts).items() if v is not None}
-        fetch_opts_dict["destination"] = chart_dir
         if config.version is not None:
             fetch_opts_dict["version"] = config.version
         fetch_opts = FetchOpts(**fetch_opts_dict)
-
-        # Fetch the chart.
-        _fetch(chart_to_fetch, fetch_opts)
-        # Sort the directories into alphabetical order, and choose the first
-        fetched_chart_name = sorted(os.listdir(chart_dir), key=str.lower)[0]
-        chart = os.path.join(chart_dir, fetched_chart_name)
+        config.fetch_opts = fetch_opts
     else:
         chart = config.path
 
-    default_values = os.path.join(chart, 'values.yaml')
+    config.release_name = release_name
 
-    # Write overrides file.
-    vals = config.values if config.values is not None else {}
-    data = pulumi.Output.from_input(vals).apply(lambda x: json.dumps(x))
```

Comment:
We're moving from working with `Output<T>` of the config values for dumping json, to `<T>` - do we expect any breaking changes?

Reply:
Good catch. This was a mistake that are causing current test failures.